perun-core/src/main/java/cz/metacentrum/perun/taskslib/dao/jdbc/TaskDaoJdbc.java

```diff
@@ -51,6 +53,18 @@ public static SimpleDateFormat getDateFormatter() {
 		return df;
 	}
 
+	/**
+	 * Method create formatter with default settings for perun timestamps and set ResolverStyle to STRICT
+	 * Timestamp format:  "dd-MM-yyyy HH:mm:ss" - "ex. 01-01-2014 10:10:10"
+	 *
+	 * ResolverStyle.STRICT means that formatter will be more strict to creating timestamp from string
+	 *
+	 * @return date formatter
+	 */
+	public static DateTimeFormatter getDateTimeFormatter() {
```

Comment:
If this suppose to replace former ``getDateFormatter()`` from above, you should replace all usages, not just a single usage in ``TaskResultDaoJdbc``. There is a plenty of usages from within ``TaskDaoJdbc`` class itself.